### ISSUE TITLE
Support for tags that aren't explicitly supported

### ISSUE THREAD

**Issue by OctaneTwisted** [jherby2k/AudioWorks#25]
Hi,

If I issue `Get-AudioFile '.\test.flac' | Export-AudioFile LameMP3 .`, tags that aren't explicitly supported are not carried over, "Disc number" and "Total discs", for instance. Nor are they visible in the metadata property of a TaggedAudioFile object.

Wouldn't it be a good idea to support all tags, including non-standard ones?

**Comment by jherby2k** [comment]
It's hard to do as the tags are mapped between audio formats, and some (like ID3) are very particular about what's allowed and how they should be formatted. I'm certainly open to adding specific additional fields like the above. Unfortunately as-is, its a breaking change. Not ideal, I agree.

**Comment by nonspin** [comment]
If for example the below Tags are present/set in the source ...
```
Disc                       : 1
DiscCount                  : 1
```
after conversion (no matter what Encoder/TagVersion/TagEncoding)
they are showing
```
Disc                       : 0
DiscCount                  : 0
```
not NULL, not Empty -> "0"

It seems like it's not setting them, because "TrackCount" is also set to "0"..
TrackCount can't be smaller than "Track", can it ? 
"Track" however is set correctly

**Comment by jherby2k** [comment]
There is no way its setting the Disc and DiscCount fields at all. What are you using to read the tags? Can you attach the file in question?

**Comment by regs01** [comment]
Sometimes there are multiple discs (volumes). Track number is given within of the volume, not of total.
There are m4a and mp3 examples. Zipped, as Guthub doesn't allow m4a and mp3 extensions.
[test.zip](https://github.com/jherby2k/AudioWorks/files/5298929/test.zip)

**Comment by jherby2k** [comment]
> Sometimes there are multiple discs (volumes). Track number is given within of the volume, not of total.
> There are m4a and mp3 examples. Zipped, as Guthub doesn't allow m4a and mp3 extensions.
> [test.zip](https://github.com/jherby2k/AudioWorks/files/5298929/test.zip)

Yes, i see that these have a disk # recorded in them. Currently, AudioWorks will strip those when re-writing the tags. I am going to address this in v1.1 by expanding the known tag dictionary. An alternative would be to simply save and restore unrecognized tags verbatim, but i'm not sure i want that feature - I like that AudioWorks re-writes the tag from scratch, as it can correct mistakes. Also that wouldn't help in a transcoding scenario.
